modules/sdk-coin-ton/src/lib/singleNominatorWithdrawBuilder.ts

```diff
@@ -0,0 +1,18 @@
+import { TransactionBuilder } from './transactionBuilder';
+import { BaseCoin as CoinConfig } from '@bitgo/statics';
+import { Recipient, TransactionType } from '@bitgo/sdk-core';
+
+export class SingleNominatorWithdrawBuilder extends TransactionBuilder {
+  constructor(_coinConfig: Readonly<CoinConfig>) {
+    super(_coinConfig);
+  }
+
+  protected get transactionType(): TransactionType {
+    return TransactionType.SingleNominatorWithdraw;
+  }
+
+  send(recipient: Recipient): SingleNominatorWithdrawBuilder {
+    this.transaction.recipient = recipient;
+    return this;
+  }
```

Comment:
you need add a method for withdrawAmount, and build message payload internally. And return error if user is trying to set a message.